scripts/release.sh

```diff
@@ -59,8 +59,8 @@ _do_release_commit() {
   $__sed -i.bk 's/(NVIM_VERSION_PRERELEASE) "-dev"/\1 ""/' CMakeLists.txt
   if grep '(NVIM_API_PRERELEASE true)' CMakeLists.txt > /dev/null; then
     $__sed -i.bk 's/(NVIM_API_PRERELEASE) true/\1 false/' CMakeLists.txt
-    build/bin/nvim --api-info > test/functional/fixtures/api_level_$__API_LEVEL.mpack
-    git add test/functional/fixtures/api_level_$__API_LEVEL.mpack
+    build/bin/nvim --api-info > test/functional/fixtures/api_level_"$__API_LEVEL".mpack
+    git add test/functional/fixtures/api_level_"$__API_LEVEL".mpack
```

Comment:
is this more readable / less error-prone? 

```suggestion
    git add "test/functional/fixtures/api_level_${__API_LEVEL}.mpack"
```